waspc/src/Wasp/Generator/ServerGenerator/JsImport.hs

```diff
@@ -0,0 +1,30 @@
+module Wasp.Generator.ServerGenerator.JsImport where
+
+import Data.Maybe (fromJust)
+import qualified StrongPath as SP
+import qualified Wasp.AppSpec.ExtImport as EI
+import Wasp.Generator.JsImport (RelDirToExternalCodeDir, mkJsImportGetterFromExtSrcDir)
+import Wasp.Generator.ServerGenerator.ExternalCodeGenerator (extServerCodeDirInServerSrcDir)
+import Wasp.JsImport
+  ( JsImport,
+    JsImportIdentifier,
+    JsImportStatement,
+  )
+import qualified Wasp.JsImport as JI
+
+-- | Wrapper function to avoid needing to import getJsImportStmtAndIdentifier from Wasp.JsImport
+--  in most cases.
+getJsImportStmtAndIdentifier ::
+  RelDirToExternalCodeDir ->
+  EI.ExtImport ->
+  (JsImportStatement, JsImportIdentifier)
+getJsImportStmtAndIdentifier relDirToExternalCodeDir = JI.getJsImportStmtAndIdentifier . getJsImport relDirToExternalCodeDir
+
+-- | Generates a JsImport from an ExtImport and relative to server ext code dir.
+getJsImport ::
```

Comment:
I would potentially rename this one to `jsImportFromExtImport`. Or `extImportToJsImport`.

Reply:
I've went with `extImportToJsImport` since it reads nicer to me. Conversion from `ExtImport` to `JsImport` :

Reply:
Ok, good job! Just a subjective reasoning why I prefer starting with the produced thing instead of the "ingredient", due to two reasons:
1. When reading code I more often care about what is function producing, in cases like this.
2. I might have multiple ways to create JsImport, so in that case I find it nice that all those functions start in a same way, e.g. with `makeJsImport` and then one can continue with `fromX`, another can be `fromY`, another can have no suffix because it creates it from scratch, ... .